### DISCUSSION TITLE
Harman AVMPlus Source Code?

### DISCUSSION THREAD

**Discussion by Fancy2209** [airsdk/Adobe-Runtime-Support#3372] [Q&A] [answered]
Adobe had the AVMPlus source code available at https://github.com/adobe/avmplus, does Harman plan on open sourcing their version used in AIR?

**Comment by ajwfrost** [comment, marked as answer]
That's still the best reference .. it's not completely updated, Adobe changed their policy on that so there are changes in AIR that aren't in the open source repo, but it's still a useful reference sometimes!

We can't open source any of the code we have under license from Adobe, the terms of the agreement don't allow that..

thanks

**Reply by Fancy2209** [reply]
Ah I see, that's unfortunate, good to know.
